StPetersburg/InEtn40559.xml

```diff
@@ -0,0 +1,224 @@
+                    <locus from="2r" n="1"></locus>
+                    <locus to="2r" n="68"></locus>
+                    <title type="complete" ref="LIT5876AsmatPrayer"></title>
+                    <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"></gap> ኤኮስ፡ አስሌ፡ ኤፓ፡</incipit>
+                  </msItem>
+                  <msItem xml:id="ms_i6">
+                    <locus from="2r" n="69"></locus>
+                    <locus to="2r" n="85"></locus>
+                    <title type="incomplete" ref="LIT2452Tergwa"><certainty locus="value" match="../@ref" cert="medium"></certainty></title>
+                    <textLang mainLang="gez"></textLang>
+                    <incipit xml:lang="gez">አሌፍ፡ ብፁዓን፡ እለ፡ ንፁሃን፡</incipit>
+                  </msItem>
+                </msContents>
+                <physDesc>
+                  <objectDesc form="Scroll">
+                      <supportDesc>
+                          <support>
+                              <material key="parchment"/>
+                              <note>A parchment strip of 14,5 cm is attached to the top of the scroll.</note>
+                          </support>
+                          <extent>
+                              <measure unit="leaf">2</measure>
+                              <dimensions type="outer" unit="mm">
+                                <height>1445</height>
+                                <width>75</width>
+                              </dimensions>
+                          </extent>
+                          <condition key="deficient">The parchement was torn and repaired with a piece of white thread.</condition>
+                      </supportDesc>
+                      <layoutDesc>
+                        <layout>First 14 lines are written in 1 column, further the text is layouted in 2 columns
+                          corresponding to the first strip of parchment.
+                        The last part is written in 1 column corresponding to the second strip of parchment.</layout>
+                        <layout corresp="#ms_i1" columns="1" writtenLines="14"></layout>
+                        <layout corresp="#ms_i1 #ms_i2 #ms_i3 #ms_i4" columns="2" writtenLines="76 115"></layout>
+                        <layout corresp="#ms_i5 #ms_i6" columns="1" writtenLines="85"></layout>
+                      </layoutDesc>
+                  </objectDesc>
+                  <handDesc>
+                    <handNote script="Ethiopic" xml:id="h1">
+                        <seg type="ink" corresp="#ms_i1 #ms_i2">Faded black, red.</seg>
+                        <seg type="ink" corresp="#ms_i3 #ms_i4 #ms_i5 #ms_i6">Black, red.</seg>
+                        <seg type="script">The script is regular according to
+                          <bibl><ptr target="bm:Chernetsov1975Svitki"/>
+                        <citedRange unit="page">223</citedRange></bibl>.</seg>
+                        <date>Mid 19th century.</date>
```

Comment:
See my remark in a previous PR - i really think that adding the attributes `<date notBefore....notAfter>` would improve on this, and if you want the paragraphic date to be there, because you prefer it, it can stay exactly where it is.